odoo/addons/base/models/res_users.py

```diff
@@ -19,7 +20,18 @@
 _logger = logging.getLogger(__name__)
 
 # Only users who can modify the user (incl. the user herself) see the real contents of these fields
-USER_PRIVATE_FIELDS = ['password']
+USER_PRIVATE_FIELDS = []
+
+DEFAULT_CRYPT_CONTEXT = passlib.context.CryptContext(
+    # kdf which can be verified by the context. The default encryption kdf is
+    # the first of the list
+    ['pbkdf2_sha512', 'plaintext'],
```

Comment:
why not keep `md5_crypt`?

Reply:
Because it was deprecated 4 years ago, and I'm not sure auth_crypt was even auto-installed at the time.